app/assets/stylesheets/govuk_publishing_components/components/_document-list.scss

```diff
@@ -43,6 +43,12 @@
   margin: 0 0 5px 0;
 }
 
+.gem-c-document-list__subtext {
+  @include govuk-font(14);
+  margin: govuk-spacing(1) 0 0 0;
```

Comment:
Could this just be `margin-top: govuk-spacing(1)` instead?

Reply:
No - the paragraph seems to be inheriting bottom margin from somewhere, which I wanted to remove.